### ISSUE TITLE
RADIUS accounting doesn't work with our current schema

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#1285]
Getting loads of errors related to the procedure on every accounting packet : 

```
Wed Mar 30 12:31:42 2016 : ERROR: (62) sql: ERROR: rlm_sql_mysql: ERROR 1054 (Unknown column 'acctstoptime' in 'field list'): 42S22
Wed Mar 30 12:31:42 2016 : ERROR: (63) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
Wed Mar 30 12:31:44 2016 : ERROR: (64) sql: ERROR: rlm_sql_mysql: ERROR 1054 (Unknown column 'acctstoptime' in 'field list'): 42S22
Wed Mar 30 12:31:44 2016 : ERROR: (65) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
Wed Mar 30 12:31:46 2016 : ERROR: (67) sql: ERROR: rlm_sql_mysql: ERROR 1054 (Unknown column 'acctstoptime' in 'field list'): 42S22
Wed Mar 30 12:31:46 2016 : ERROR: (66) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
Wed Mar 30 12:31:48 2016 : ERROR: (69) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
Wed Mar 30 12:31:48 2016 : ERROR: (68) sql: ERROR: rlm_sql_mysql: ERROR 1054 (Unknown column 'acctstoptime' in 'field list'): 42S22
Wed Mar 30 12:31:50 2016 : ERROR: (70) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
Wed Mar 30 12:31:52 2016 : ERROR: (71) sql: ERROR: rlm_sql_mysql: ERROR 1318 (Incorrect number of arguments for PROCEDURE pf.acct_start; expected 23, got 22): 42000
```

**Comment by louismunro** [comment]
Working on it.
I think we just need to update the schema to reflect the same changes that were introduced in the upgrade.sql script.

**Comment by louismunro** [comment]
Should be fixed by 5fcefb515a6081bd41bdaebeb3cdd8d9d14aa0cd 
Please test.

**Comment by julsemaan** [comment]
message indeed not appearing anymore

I'm sure its linked to your commit, just more like RADIUS black magic
